app/jobs/daily_broadcast_auctions_job.rb

```diff
@@ -0,0 +1,19 @@
+class DailyBroadcastAuctionsJob < ApplicationJob
+  def perform
+    User.where(daily_summary: true).each do |user|
```

Comment:
The knowledge that a `User` model has `daily_summary` model belongs to a model. I realize that `ActiveRecord` pattern (unfortunately) encourages to treat a model more like a data bag instead of real object, but perhaps it still would be better to keep that knowledge in the model, when possible?

Something like `User.subscribed_to_daily_summary`.